pyinter/interval.py

```diff
@@ -203,10 +203,13 @@ def empty(self):
 
     def overlaps(self, other):
         """If self and other have any overlaping values returns True, otherwise returns False"""
-        if self.lower_value in other or self._upper_value in other or \
-                other.lower_value in self or other._upper_value in self or self == other:
-            return True
-        return False
+        if self > other:
+            self, other = other, self
```

Comment:
Yeah, originally I was calling `other.overlaps(self)` but then I realized that that was completely unnecessary :P